module/strategy/StartSafely/src/StartSafely.cpp

```diff
@@ -0,0 +1,92 @@
+#include "StartSafely.hpp"
+
+#include "extension/Configuration.hpp"
+
+#include "message/actuation/Limbs.hpp"
+#include "message/actuation/ServoCommand.hpp"
+#include "message/input/Sensors.hpp"
+#include "message/strategy/StartSafely.hpp"
+
+
+namespace module::strategy {
+
+    using StartSafelyTask = message::strategy::StartSafely;
+    using message::actuation::Body;
+    using message::actuation::ServoCommand;
+    using message::actuation::ServoState;
+    using message::input::Sensors;
+
+    using extension::Configuration;
+
+    StartSafely::StartSafely(std::unique_ptr<NUClear::Environment> environment)
+        : BehaviourReactor(std::move(environment)) {
+
+        on<Configuration>("StartSafely.yaml").then([this](const Configuration& config) {
+            // Use configuration here from file StartSafely.yaml
+            this->log_level = config["log_level"].as<NUClear::LogLevel>();
+
+            cfg.move_time   = config["move_time"].as<double>();
+            cfg.servo_gain  = config["servo_gain"].as<double>();
+            cfg.servo_error = config["servo_error"].as<double>();
+
+            cfg.servo_targets[0]  = config["right_shoulder_pitch"].as<double>();
+            cfg.servo_targets[1]  = config["left_shoulder_pitch"].as<double>();
+            cfg.servo_targets[2]  = config["right_shoulder_roll"].as<double>();
+            cfg.servo_targets[3]  = config["left_shoulder_roll"].as<double>();
+            cfg.servo_targets[4]  = config["right_elbow"].as<double>();
+            cfg.servo_targets[5]  = config["left_elbow"].as<double>();
+            cfg.servo_targets[6]  = config["right_hip_yaw"].as<double>();
+            cfg.servo_targets[7]  = config["left_hip_yaw"].as<double>();
+            cfg.servo_targets[8]  = config["right_hip_roll"].as<double>();
+            cfg.servo_targets[9]  = config["left_hip_roll"].as<double>();
+            cfg.servo_targets[10] = config["right_hip_pitch"].as<double>();
+            cfg.servo_targets[11] = config["left_hip_pitch"].as<double>();
+            cfg.servo_targets[12] = config["right_knee"].as<double>();
+            cfg.servo_targets[13] = config["left_knee"].as<double>();
+            cfg.servo_targets[14] = config["right_ankle_pitch"].as<double>();
+            cfg.servo_targets[15] = config["left_ankle_pitch"].as<double>();
+            cfg.servo_targets[16] = config["right_ankle_roll"].as<double>();
+            cfg.servo_targets[17] = config["left_ankle_roll"].as<double>();
+            cfg.servo_targets[18] = config["head_yaw"].as<double>();
+            cfg.servo_targets[19] = config["head_pitch"].as<double>();
```

Comment:
```
for (uint8_t i  = 0; i<config["servo_targets"].size(); ++i) {
    const auto& servo_name = config[i].first; // For the sake of clarity
    const auto& servo_position = config[i].second;
    cfg.servo_targets[i] = servo_position;
}
```

then servo_targets is just a sequence of map entries, or alternatively something that will make this code a lot shorter